src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -577,6 +587,18 @@ internal void Save(string filePath, bool isBackup = false, EngineController engi
                 throw (ex);
             }
         }
+        /// <summary>
+        /// This function appends view block to the model json
+        /// </summary>
+        /// <param name="data">Workspace Model data in JSON format</param>
+        private JObject AddViewBlockToJSON(string modelData)
+        {
+            var jo = JObject.Parse(modelData);
```

Comment:
maybe check for parsing failures here - consider someone calling this with any string...

Reply:
well I can do that. but this is been like this for a long time.